docker/README.md

```diff
@@ -82,8 +82,24 @@ To support running the services locally using docker, there are three different
   > Building the containers in production mode takes longer as all node modules need to be installed and additional build
   > steps such as compiling SASS need to be performed.
 
-In order to run the services locally, you'll need to insert the appropriate values into the environment files ending with .secrets.env
-in the [env](env) folder.
+In order to run the services locally, you'll need to to rename the env files in the in the [env](env) folder to include a leading dot and removing .example. You'll need to insert the appropriate values into the environment files ending with .secrets.env. Either rename as below or grab the filles from the the gitlab repository: fish/rod-licensing-env-vars
+
+To rename the files:
+
+```shell script
+git mv fulfilment_job.env.example .fulfilment_job.env
+git mv fulfilment_job.secrets.env.example .fulfilment_job.secrets.env
+git mv gafl-webapp_telesales.env.example .gafl-webapp_telesales.env
+git mv gafl-webapp_telesales.secrets.env.example .gafl-webapp_telesales.secrets.env
+git mv gafl-webapp.env.example .gafl-webapp.env
+git mv gafl-webapp.secrets.env.example .gafl-webapp.secrets.env
+git mv payment-mop_up_job.env.example .payment-mop_up_job.env
+git mv payment-mop_up_job.secrets.env.example .payment-mop_up_job.secrets.env
+git mv pocl_job.env.example .pocl_job.env
+git mv sales_api.env.example .sales_api.env
+git mv sales_api.secrets.env.example .sales_api.secrets.env
+git mv sqs_receiver.env.example .sqs_receiver.env
+```
```

Comment:
You don't want to do `git mv` as that'd rename them within git. `cp fulfilment_job.env.example .fulfilment_job.env`, etc, would do what's needed as we want to keep the original